### PR TITLE
Firmware and quality management standard

### PR DESCRIPTION
**Pull request by JasonBrave**

### CODE REVIEW

File: general/standards/electrical-software-quality-management.rst [hunk 1]
```diff
@@ -1,2 +1,22 @@
 Electrical & Software Quality Management Process
 ================================================
+
+PCB designs
+-----------
+
+* Minimum 4 approves for schematics review
```
Comment (Joe-Joe-Joe-Joe) [inline general/standards/electrical-software-quality-management.rst:7]: This is a bit much no? This was already a huge bottleneck last year to the point where it got ignored.

File: general/standards/electrical-software-quality-management.rst [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+* Minimum 4 approves for PCB layout
+* Minimum 4 approves for routed PCB
+* For each of the 4 reviewers mentioned above, at minimum 2 need to have experience on designing PCB for Waterloo Rocketry team
+* All electrical standard need to be followed, and any exception standard must be explicitly noted
+
+Firmware (Embedded Software)
+----------------------------
+
+* Each PR need at minimum 2 approve to get merged
+* One of the approve have to come from CODEOWNERS (usually project owner)
```
Comment (Joe-Joe-Joe-Joe) [inline general/standards/electrical-software-quality-management.rst:17]: A significant number of PRs will likely be coming from the project owner. For smaller boards that might only be one person. I don't think this is practical.

File: general/standards/electrical-standard.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Electrical Standard
+===================
+
+Refer to this `Google Doc <https://docs.google.com/document/d/1Dor3P09H9-mdcJvkMIZcK-YoCY1UyPBDU1qtubGDoAM/edit>`_
```
Comment (Joe-Joe-Joe-Joe) [inline general/standards/electrical-standard.rst:4]: Can we pleeeeeeeease copy the contents of the doc onto the site? For me?

File: general/standards/embedded-coding-standard.rst [hunk 4]
```diff
@@ -1,2 +1,46 @@
 Embedded (Firmware) Coding Standard
-===================================
+***********************************
```
Comment (Joe-Joe-Joe-Joe) [inline general/standards/embedded-coding-standard.rst:2]: Does all embedded code have to be in C? The others were playing with the idea of C and I'd rather this be documented than me linking that one thread every time.

File: general/standards/electrical-software-quality-management.rst (hunk 2 above)
Comment (celery6) [reply general/standards/electrical-software-quality-management.rst:17]: maybe this indicates a different issue, that projects shouldn't only have 1 codeowner...? even for small boards, i'd think its always better to have at least 2 people responsible.

File: general/standards/electrical-standard.rst (hunk 3 above)
Comment (JasonBrave) [reply general/standards/electrical-standard.rst:4]: Will fix in future PR

**Pull request merged**